ui_framework/doc_site/src/views/form/form_example.js

```diff
@@ -28,6 +28,19 @@ export default props => (
         code: labelHtml,
       }]}
     >
+
+      <GuideText>
+        Never forget to label every input element. You can either
+        use a <code>label</code> element with a <code>for</code> attribute
+        referencing the <code>id</code> of the input field, wrap the <code>input</code> field
+        within the <code>label</code> element or use <code>aria-label</code> or <code>aria-labelledby</code>.
+      </GuideText>
+
+      <GuideText>
+        For the sake of simplicity we haven&rsquo;t labeled the element on this page
```

Comment:
Is this meant to refer to the `<label>` example or to all of the different input examples on the page?